src/Symfony/Component/Mailer/Tests/EventListener/DkimSignedMessageListenerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Tests\EventListener;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mailer\Envelope;
+use Symfony\Component\Mailer\Event\MessageEvent;
+use Symfony\Component\Mailer\EventListener\DkimSignedMessageListener;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Crypto\DkimSigner;
+use Symfony\Component\Mime\Header\Headers;
+use Symfony\Component\Mime\Header\MailboxListHeader;
+use Symfony\Component\Mime\Message;
+use Symfony\Component\Mime\Part\TextPart;
+
+class DkimSignedMessageListenerTest extends TestCase
+{
+    public function testDkimMessageSigningProcess()
+    {
+        if (!\extension_loaded('openssl')) {
```

Comment:
use `@requires extension openssl` on the test instead (in both tests), for consistency with the way we handle those checks in tests